common/network_manager.go

```diff
@@ -254,13 +254,13 @@ func (m *NetworkManager) emitResponseMetrics(resp *Response, req *Request) {
 				// issues with the parsing and conversion to `time.Time`.
 				// Have not spent time looking for the root cause of this in the Chromium source to file a bug report, and neither
 				// Puppeteer nor Playwright seems to care about the `responseTime` value and don't use/expose it.
-				Value: k6metrics.D(timestamp.Sub(req.timestamp)),
-				Time:  timestamp,
+				Value: k6metrics.D(wallTime.Sub(req.wallTime)),
+				Time:  wallTime,
```

Comment:
> I think we should keep it, just confirm whether the `event.Timestamp.Time()` it uses is correct for the browser metrics, or if we should also apply a similar offset as we do here.
> 
> EDIT: Actually, on closer look, both `Frame.initTime` and `event.Timestamp` are monotonic values, and since [the value is calculated for each sample](https://github.com/grafana/xk6-browser/blob/508bcb861edcdad7615e53cbf64c3829b4dfd0b3/common/frame.go#L342-L343), this is the correct usage of `event.Timestamp`, and the value should also be correct. But feel free to confirm to make sure.

Thanks, @imiric, I'll check this out.